dist/sonatype_nexus_upload.py

```diff
@@ -72,20 +72,20 @@ def _install_locally(version, files):
         shutil.copyfile(file, os.path.join(path, basename))
 
 
-def _urlopen_retried(request, max_retries=20, attempt=1, delay_sec=1):
+def _urlopen_retried(request, max_retries=20, attempt=1):
     """
     Retries a request via recursion. Retries happen with a default delay of 1 second. We do not exponentially back off.
     :param request: the request to be made
     :param max_retries: Number of retries to use, default is 20. The reason we are using such a high retry is because
     sonatype fails quite frequently
     :param attempt: The current attempt number for the request
-    :param delay_sec: The delay before making a retried request
     :return: the response if successful, raises error otherwise
     """
     try:
         return urlopen(request)
     except HTTPError as e:
-        if max_retries > 0 and e.code >= 500:
+        if max_retries > attempt and e.code >= 500:
+            delay_sec = 1
```

Comment:
Why not keep this in the params?

Reply:
It's never set to anything other than 1 and the docstrings in place say this function always retries after `1s`

Reply:
That's as a default value though. Would you be against keeping this in the parameter? It seems a little easier to see where this value is configured

Reply:
updated

Reply:
Thanks, @rebello95